gravitee-apim-console-webui/src/management/application/details/subscribe/application-subscribe.controller.ts

```diff
@@ -49,6 +49,11 @@ class ApplicationSubscribeController {
 
   async $onInit() {
     const subscriptionsByAPI = _.groupBy(this.subscriptions.data, 'api');
+
+    await this.ApiService.list(null, true, null, null, null, Object.keys(subscriptionsByAPI)).then(
```

Comment:
A bit strange to mix `await` and `.then`
Could be replaced by something like this:
```suggestion
    this.apis = (await this.ApiService.list(null, true, null, null, null, Object.keys(subscriptionsByAPI))).data
```